File: iniciante/1010/1010.js
var input = require("fs").readFileSync("dev/stdin", "utf8");
var lines = input.split("\n");

var produto1 = lines.shift().split(" ");
var produto2 = lines.shift().split(" ");

var total = produto1[1] * produto1[2] + produto2[1] * produto2[2];

console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`);

File: iniciante/1010/1010.py
# -*- coding: utf-8 -*-

produto1 = input().split(' ')
produto2 = input().split(' ')

total = float(produto1[1]) * float(produto1[2]) + \
    float(produto2[1]) * float(produto2[2])

print(f'VALOR A PAGAR: R$ {total:.2f}')
